frontend/src/util/index.ts

```diff
@@ -224,3 +224,15 @@ export const useNoindexTag = (noindex = true) => {
         };
     });
 };
+
+/** Formats the given number of milliseconds as ISO 8601 duration string, e.g. "PT3M47S" */
+export const toIsoDuration = (milliseconds: number): string => {
+    let acc = milliseconds / 1000;
+    const seconds = acc % 60;
+    acc /= 60;
```

Comment:
Don't you need to `Math.floor` here? :thinking: Maybe not because of the `%`? Not sure, though.